NEWS.rst

```diff
@@ -2,6 +2,60 @@
 NEWS
 ******
 
+Version 1.0-rc4
+===============
+
+:date: 2018-01-18
+:commit: nil
+
+Changes
+-------
+
+* Multi stage Docker builds to use prebuilt Envoy dependencies. (#2452, @jrajahalme)
```

Comment:
Sorry I also noticed the link to the PR is not working since rc2. Can you replace all `#ID` with `ID_`?

Reply:
fixed. This is still a manual sed for now and not automated